// @warlords/common/dist/game/model/enum/resource_type.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.ResourceType = void 0;
var ResourceType;
(function (ResourceType) {
    ResourceType["Food"] = "food";
    ResourceType["Wood"] = "wood";
    ResourceType["Iron"] = "iron";
    ResourceType["Stone"] = "stone";
    ResourceType["notApply"] = "notapply";
    // Buffs for level
    ResourceType["BuffFood"] = "bufffood";
    ResourceType["BuffWood"] = "buffwood";
    ResourceType["BuffIron"] = "buffiron";
    ResourceType["BuffStone"] = "buffstone";
    ResourceType["BuffTimeUnit"] = "bufftimeunit";
    ResourceType["BuffTimeUnitBuilding"] = "bufftimeunitbuilding";
    ResourceType["BuffTimeSearch"] = "bufftimesearch";
    ResourceType["BuffStorage"] = "buffstorage";
    ResourceType["BuffPopulation"] = "buffpopulation";
    ResourceType["BuffTransaction"] = "bufftransaction";
    ResourceType["BuffPopulationHappiness"] = "buffpopulationhappiness";
})(ResourceType = exports.ResourceType || (exports.ResourceType = {}));
