// @warlords/common/dist/game/index.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.ResourceType = exports.ProvinceType = exports.UnitType = exports.NpcType = exports.BuildingType = exports.province_scale_default = exports.building_scale_default = void 0;
// config default
var model_1 = require("./model");
Object.defineProperty(exports, "building_scale_default", { enumerable: true, get: function () { return model_1.building_scale_default; } });
var model_2 = require("./model");
Object.defineProperty(exports, "province_scale_default", { enumerable: true, get: function () { return model_2.province_scale_default; } });
// Enum's
var model_3 = require("./model");
Object.defineProperty(exports, "BuildingType", { enumerable: true, get: function () { return model_3.BuildingType; } });
var model_4 = require("./model");
Object.defineProperty(exports, "NpcType", { enumerable: true, get: function () { return model_4.NpcType; } });
var model_5 = require("./model");
Object.defineProperty(exports, "UnitType", { enumerable: true, get: function () { return model_5.UnitType; } });
var model_6 = require("./model");
Object.defineProperty(exports, "ProvinceType", { enumerable: true, get: function () { return model_6.ProvinceType; } });
var model_7 = require("./model");
Object.defineProperty(exports, "ResourceType", { enumerable: true, get: function () { return model_7.ResourceType; } });
